android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt

```diff
@@ -67,6 +67,7 @@ data class QuestionnaireConfig(
   val uniqueIdAssignment: UniqueIdAssignmentConfig? = null,
   val linkIds: List<LinkIdConfig>? = null,
   val showSubmitAnywayButton: String = "false",
+  val showSubmissionConfirmationDialog: String = "false",
```

Comment:
This is a new configurable feature, please add to/update the documentation to reflect this config and usage on the docusite - https://docs.opensrp.io/engineering/app/configuring/forms

Reply:
Oh, right, I'll do that